secrets/embedding.env.tpl

```diff
@@ -0,0 +1,3 @@
+EMBEDDING_MODEL=mixedbread-ai/mxbai-embed-large-v1
+EMBEDDING_QUERY_PREFIX="Represent this sentence for searching relevant passages: "
```

Comment:
What does this do? Just curious!

Reply:
Oh, this is the retrieval instruction for the embedding model. See: https://huggingface.co/mixedbread-ai/mxbai-embed-large-v1. Some models claim to perform better when an instruction used during the model training is used.